dotnet/src/IntegrationTests/Connectors/Memory/AzureAISearch/CommonKeywordVectorizedHybridSearchTests.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.Extensions.VectorData;
+using Microsoft.SemanticKernel.Connectors.AzureAISearch;
+using Xunit;
+
+namespace SemanticKernel.IntegrationTests.Connectors.Memory.AzureAISearch;
+
+/// <summary>
+/// Inherits common integration tests that should pass for any <see cref="IKeywordVectorizedHybridSearch{TRecord}"/>.
+/// </summary>
+/// <param name="fixture">Azure AI Search setup and teardown.</param>
+[Collection("AzureAISearchVectorStoreCollection")]
+[AzureAISearchConfigSetCondition]
+public class CommonKeywordVectorizedHybridSearchTests(AzureAISearchVectorStoreFixture fixture) : BaseKeywordVectorizedHybridSearchTests<string>
```

Comment:
nit: Since this test class is related to Azure AI Search only, should we be more specific in name here?

```suggestion
[Collection("AzureAISearchVectorStoreCollection")]
[AzureAISearchConfigSetCondition]
public class AzureAISearchKeywordVectorizedHybridSearchTests(AzureAISearchVectorStoreFixture fixture) : BaseKeywordVectorizedHybridSearchTests<string>
```

Reply:
I was on the fence myself, so happy to switch to this name.